src/main/msp/msp_protocol.h

```diff
@@ -214,6 +214,9 @@
 
 #define MSP_CAMERA_CONTROL              98
 
+#define MSP_BEEPER_CONFIG               99
+#define MSP_SET_BEEPER_CONFIG           100
```

Comment:
OK,  I need to choose another set, 101 is already taken by outgoing message MSP_STATUS.